### PR TITLE
Feature/donation

### PR DESCRIPTION
**Pull request by jin5578**
### Issue id
관련된 이슈를 링크해주세요.
#10 

### Why did you make these changes?

* Donation 화면 구현했습니다.

### What did you changed?

* Donation 화면 (Not Api) 구현했습니다

### What do you especially want to get reviewed?

* None

### Is there any other comments that every teammate should know?

* None

### Checklist

* [x] Gradle task: connectedCheck
* [x] Gradle task: test
* [x] Code formatting

### CODE REVIEW

**Comment by koba0909** [comment]
확인했습니다~

**Review by koba0909 — APPROVED** [review]

**Comment by Huijiny** [comment]
checklist 체크해야할것같아요~!

**Review by Huijiny — APPROVED** [review]

File: app/src/main/java/com/mashup/mobalmobal/ui/donate/DonateFragment.kt [hunk 1]
```diff
@@ -1,21 +1,37 @@
...
+        binding.tvDonate.setOnClickListener {
+            val price = binding.etDonatePrice.text.toString()
+            donateViewModel.requestDonation(price)
+        }
+    }
+
+    override fun onBindViewModels() {
+        super.onBindViewModels()
+    }
+
```
Comment (mkSpace) [inline app/src/main/java/com/mashup/mobalmobal/ui/donate/DonateFragment.kt]: 지워도 될 것 같아요

File: app/src/main/res/layout/fragment_donate.xml [hunk 2]
```diff
@@ -2,15 +2,100 @@
...
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guide_end_20"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_end="20dp" />
+
+    <ImageView
+        android:id="@+id/iv_previous"
```
Comment (mkSpace) [inline app/src/main/res/layout/fragment_donate.xml]: 저희 view 형식에 따라 iv, tv등 붙이기로 했었나요??

File: app/src/main/java/com/mashup/mobalmobal/ui/donate/DonateViewModel.kt [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package com.mashup.mobalmobal.ui.donate
+
+import android.util.Log
```
Comment (mkSpace) [inline app/src/main/java/com/mashup/mobalmobal/ui/donate/DonateViewModel.kt]: Optimize imports 해주세요

**Review by mkSpace — APPROVED** [review]
GOOD!

**Pull request merged**